src/containers/Cfe/index.tsx

```diff
@@ -128,103 +130,146 @@ class Cfe extends React.Component<CfeProps, CfeState> {
+                            onDismissCheckboxChecked={this.onDismissCheckboxChecked}
+                            visible={showSmallScreenWarning}
                         />
-                        {this.props.showAlignControl && (
-                            <AlignControl
-                                parent={this.alignContainer}
-                                aligned={this.props.alignActive}
-                                setAligned={this.props.setAlignActive}
+                        <Content className={plotClassNames}>
+                            <PlotTab />
+                        </Content>
+                        <Content className={viewerClassNames}>
+                            <CellViewer
+                                onControlPanelToggle={this.onControlPanelToggle}
+                                {...this.props.volumeViewerProps}
                             />
-                        )}
-                    </Content>
+                            {this.props.showAlignControl && (
+                                <AlignControl
+                                    parent={this.alignContainer}
+                                    aligned={this.props.alignActive}
+                                    setAligned={this.props.setAlignActive}
+                                />
+                            )}
+                        </Content>
+                    </Layout>
                 </Layout>
+                <Sider
+                    style={{
+                        right: "0",
+                        width: "100px",
+                        position: "absolute",
+                        backgroundColor: PALETTE.galleryBackground,
+                    }}
+                    width="100%"
+                    collapsible={true}
+                    collapsed={galleryCollapsed}
+                    onCollapse={toggleGallery}
+                    defaultCollapsed={true}
+                    collapsedWidth={100}
+                    className={styles.sider}
+                    reverseArrow={true}
+                >
+                    <ThumbnailGallery
+                        collapsed={galleryCollapsed}
+                        toggleGallery={toggleGallery}
+                        openViewerTab={() => this.setState({ currentTab: VIEWER_TAB_KEY })}
+                    />
+                </Sider>
             </Layout>
         );
+
+        return (
+            <ConfigProvider
+                theme={{
+                    components: {
+                        Layout: {
+                            headerHeight: 56,
+                        },
+                        Checkbox: {
+                            colorPrimary: PALETTE.mediumDarkGray,
+                            colorPrimaryHover: PALETTE.mediumGray,
+                        },
+                    },
+                }}
+            >
+                {layout}
+            </ConfigProvider>
```

Comment:
Wraps the final layout in a ConfigProvider to change a few specific properties (header height, checkbox colors)